x/consortium/keeper/keys.go

```diff
@@ -0,0 +1,24 @@
+package keeper
+
+import (
+	sdk "github.com/line/lbm-sdk/types"
+)
+
+// Keys for consortium store
+// Items are stored with the following key: values
+//
+// - 0x00: Params
+//
+// - 0x01<valAddress_Bytes>: bool
+var (
+	paramsKey              = []byte{0x00}
+	validatorAuthKeyPrefix = []byte{0x01}
+)
+
+// validatorAuthKey key for a specific validator from the store
+func validatorAuthKey(valAddr sdk.ValAddress) []byte {
+	key := make([]byte, len(validatorAuthKeyPrefix)+len(valAddr))
+	copy(key, validatorAuthKeyPrefix)
+	copy(key[len(validatorAuthKeyPrefix):], valAddr)
+	return key
+}
```

Comment:
This function doesn't check if this key is multisig or not. 
Do you add this validating function later?

Reply:
It has nothing to do with the new feature. 
The function would be implemented on another issue.